src/main/java/com/dutact/web/features/event/admin/dtos/EventCreateDto.java

```diff
@@ -1,33 +1,34 @@
 package com.dutact.web.features.event.admin.dtos;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotNull;
 import lombok.Data;
+import org.springframework.web.multipart.MultipartFile;
 
 import java.time.LocalDateTime;
 
 @Data
 public class EventCreateDto {
-    @JsonProperty("organizerId")
+    @NotNull
```

Comment:
why u change from json property to jarkartar validation annotation

Reply:
@ModelAttribute does not use json